### ISSUE TITLE
Newpct change and the plugin dont work-Unable to locate torrent ID 

### ISSUE THREAD

**Issue by davidcsgf** [Flexget/Flexget#1964]
### Expected behaviour:

Download torrent from newpct, thread has been created in the forum:
https://discuss.flexget.com/t/newpct-plugin-obsolete/3733

### Steps to reproduce:
- Step 1: Download series form Newpct

#### Config:
```
https://pastebin.com/qssgM3Qv
```
  
#### Log:
```
2017-09-14 15:21 VERBOSE newpct tv-spanish Newpct1 URL: http://newpct1.com/descargar-serie/gotham/capitulo-35/hdtv/
2017-09-14 15:22 WARNING urlrewriter tv-spanish URL rewriting newpct failed: Unable to locate torrent ID from url http://newpct1.com/descarga-torrent/descargar-serie/gotham/capitulo-35/hdtv/
```

### Additional information:
The correct torrent is in this javascript function, the value of window.location.href:
 function openTorrent()
 {
    var link = "http://advserver.xyz/v2/gena?gid=OVKYGAW2FN&uid=164";
    window.open(link);
   window.location.href = "http://tumejorjuego.com/descargar/index.php?link=descargar-torrent/98835_gotham-temporada-3-hdtv-cap-305-espaa-a-ol-castellano/";	
 }	

- FlexGet version: 2.10.89 (2017-09-11)
- Python version: 2.7.9 
- Installation method: pip
- Using daemon (yes/no): no
- OS and version: raspbian latest version

**Comment by ciltocruz** [comment]
Hahaha, I came to leave a ticket here too.

**Comment by davidcsgf** [comment]
@rsalas 
@malkavi 
@javichu666

Any idea to solve this problem??

**Comment by davidcsgf** [comment]
Thanks @cvium

**Comment by ciltocruz** [comment]
It works!! Thanks!! :+1:

**Comment by olgaoton** [comment]
Thanks!!! It works for me too

**Comment by OSTNeoX** [comment]
Hello. First, sorry for my english... :(

I'm having problems with posted yml file. 

I have set identified_by: ep and start: 3x18 on Gotham entry (this is the last episode un web today)

When i execute flexget with trace i get this results:

`2017-11-02 12:05 DEBUG    newpct        tv-spanish      Search NewPCT
2017-11-02 12:05 DEBUG    newpct        tv-spanish      Searching NewPCT Gotham S03E18
2017-11-02 12:05 DEBUG    utils.requests tv-spanish      Fetching URL http://www.newpct1.com/buscar with args () and kwargs {'data': {'q': b'Gotham S03E18'}, 'json': None, 'timeout': 30}
2017-11-02 12:05 DEBUG    newpct        tv-spanish      Searching NewPCT Gotham 3x18
2017-11-02 12:05 DEBUG    utils.requests tv-spanish      Fetching URL http://www.newpct1.com/buscar with args () and kwargs {'data': {'q': b'Gotham 3x18'}, 'json': None, 'timeout': 30}
2017-11-02 12:05 DEBUG    newpct        tv-spanish      Finish search NewPCT with 0 entries
2017-11-02 12:05 DEBUG    discover      tv-spanish      No results from newpct
2017-11-02 12:05 VERBOSE  discover      tv-spanish      No search results for `Gotham S03E18``

Yesterday newpct webpage (not newpct1) have been changed with newpct1 style. And with postman I need to add all headers to retrieve results, else i have 0 results for q: Gotham query.
